components/feature/sendtab/src/test/java/mozilla/components/feature/sendtab/SendTabUseCasesTest.kt

```diff
@@ -0,0 +1,254 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ *  License, v. 2.0. If a copy of the MPL was not distributed with this
+ *  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package mozilla.components.feature.sendtab
+
+import mozilla.components.concept.sync.ConstellationState
+import mozilla.components.concept.sync.Device
+import mozilla.components.concept.sync.DeviceCapability
+import mozilla.components.concept.sync.DeviceConstellation
+import mozilla.components.concept.sync.DeviceType
+import mozilla.components.concept.sync.OAuthAccount
+import mozilla.components.concept.sync.TabData
+import mozilla.components.service.fxa.manager.FxaAccountManager
+import mozilla.components.support.test.any
+import mozilla.components.support.test.eq
+import mozilla.components.support.test.mock
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+import org.mockito.Mockito.`when`
+import org.mockito.Mockito.never
+import org.mockito.Mockito.times
+import org.mockito.Mockito.verify
+import java.util.UUID
+
+class SendTabUseCasesTest {
+
+    private val manager: FxaAccountManager = mock()
+    private val account: OAuthAccount = mock()
+    private val constellation: DeviceConstellation = mock()
+    private val state: ConstellationState = mock()
+
+    @Before
+    fun setup() {
+        `when`(manager.authenticatedAccount()).thenReturn(account)
```

Comment:
nit: A nice way will be [whenever](https://github.com/mozilla-mobile/android-components/blob/6341e10cde7bc25f6655e3c3cd84dc6efea9fd72/components/support/test/src/main/java/mozilla/components/support/test/Mock.kt#L30) :)

Reply:
I don't follow the value of the alias:
```kotlin
whenever(manager.authenticatedAccount()).thenReturn(account)
`when`(manager.authenticatedAccount()).thenReturn(account)
```

They both do the same thing but nothing is really gained from the alias. 🤔

Reply:
Yes you're right, It's a bit easier to write because you can use the IDE auto complete, but a part from that it's just aesthetics